docs/learn-phragmen.md

```diff
@@ -526,18 +526,68 @@ their candidates selected will also not have any stake in any of the elected can
 
 ## Optimizations
 
-The results are further optimized to more evenly distribute the load (which is a desirable feature,
-as discussed below) via post-processing.
+The results for nominating validators are further optimized for several purposes:
 
-### Rationale
+1. To reduce the number of edges, i.e. to minimize the number of validators any nominator selects
+2. To ensure, as much as possible, an even distribution of stake among the validators
+3. Reduce the amount of block computation time
+
+### High-Level Description
+
+After running the weighted Phragmén algorithm, a process is run which redistributes the vote amongst
+the elected set. This process will never add or remove an elected candidate from the set. Instead,
+it reduces the variance in the list of backing stake from the voters to the elected candidates.
+Perfect equalization is not always possible, but the algorithm attempts to equalize as much as
+possible. It then runs an edge-reducing algorithm to minimize the number of validators per
+nominator, ideally giving every nominator a single validator to nominate per era.
+
+To minimize block computation time, the staking process is run as an
+[off-chain worker](https://substrate.dev/docs/en/conceptual/core/off-chain-workers). In order to
+give time for this off-chain worker to run, staking commands (bond, nominate, etc.) are not allowed
+in the last quarter of each era.
+
+These optimizations will not be covered in-depth on this page. For more details, you can view the
+[Rust implementation of elections in Substrate](https://github.com/paritytech/substrate/blob/master/frame/elections-phragmen/src/lib.rs),
+the
+[Rust implementation of staking in Substrate](https://github.com/paritytech/substrate/blob/master/frame/staking/src/lib.rs),
+or the `seqPhragménwithpostprocessing` method in the
+[Python reference implementation](https://github.com/w3f/consensus/tree/master/NPoS). If you would
+like to dive even more deeply, you can review the
+[W3F Research Page on Sequential Phragmén Method](https://research.web3.foundation/en/latest/polkadot/NPoS/4.%20Sequential%20Phragm%C3%A9n%E2%80%99s%20method.html).
+
+### Rationale for Minimizing the Number of Validators Per Nominator
+
+Paying out rewards for staking from every validator to all of their nominators can cost a
+non-trivial amount of chain resources (in terms of space on chain and resources to compute). Assume
+a system with 200 validators and 1000 nominators, where each of the nominators has nominated 10
+different validators. Payout would thus require `1000 * 10`, or 10,000 transactions. In an ideal
+scenario, if every nominator selects a single validator, only 1,000 transactions would need to take
+place - an order of magnitude fewer. Empirically, network slowdown at the beginning of an era has
+been seen due to the large number of individual payouts by validators to nominators. In extreme
```

Comment:
Consider changing from passive "has been seen"

Reply:
Changed to "occurred"